reference/CloudPay-API-Specification.yaml

```diff
@@ -1381,6 +1381,115 @@ paths:
 
         **Note** *Only one of the four authorisation parameters are required*
     parameters: []
+  /api/v1/account/profile/billing:
+    put:
+      summary: Update the customer's Stripe billing profile
+      operationId: put-api-v1-account-profile-billing
+      parameters:
+        - $ref: '#/components/parameters/apiJwtTokenParam'
+        - $ref: '#/components/parameters/apiSessionIdParam'
+        - $ref: '#/components/parameters/thirdPartyAuthCookie'
+        - $ref: '#/components/parameters/authCookie'
+      tags:
```

Comment:
We're getting 'operation tags must be defined in global tags' warning, AFAIK the tag is used to group operations into one category, so we can specify one tag here OR specify new tags in global tag list - but this second option may cause confusion with same operation in multiple categories.